addon/-private/data-view/utils/scroll-handler.js

```diff
@@ -36,27 +36,23 @@ export class ScrollHandler {
         handlers
       };
       // TODO add explicit test
-      if (SUPPORTS_PASSIVE) {
+      if (this.isUsingPassive) {
```

Comment:
in the constructor there is

```js
this.isUsingPassive = SUPPORTS_PASSIVE;
```

I guess, this way there is a chance to configure  `isUsingPassive` in tests.